main/static/autocom.js

```diff
@@ -267,9 +284,13 @@ function generateTooltipContents(tooltip, words, index, selectedCallback) {
 }
 
 // Inserts `word` at the location specified by `at` inside of `input`.
-function insertWord(input, at, word) {
+function insertWord(input, at, word, supressCallback) {
 	input.value = input.value.substring(0, at.from) + word + input.value.substring(at.to);
-	input.selectionStart = input.selectionEnd = at.from + word.length;
+	setTimeout(function() {
+		input.focus();
+		input.selectionStart = input.selectionEnd = at.from + word.length;
+		supressCallback();
```

Comment:
I found that when you clicked on a autocomplete choice (instead of using tab/enter to fill it in), then the input box would unselected.

The call to `.focus()` brings the input back into focus. Assigning the `.selectionStart` causes the cursor to be at the end of the inserted word (instead of wherever it was before or at the end of the input).

The `suppressCallback()` has the effect of suppressing the tooltip until further input- like when you press enter to autocomplete, the suggestions go away (until you type another character).